perfkitbenchmarker/providers/gcp/gcp_dataproc.py

```diff
@@ -65,13 +67,37 @@ def _Exists(self):
     stdout, stderr, retcode = cmd.Issue()
     return retcode == 0
 
-  def SubmitJob(self, jarfile, classname, job_poll_interval=None):
+  def SubmitJob(self, jarfile, classname, job_poll_interval=None,
+                job_arguments=None, job_stdout_file=None):
     cmd = util.GcloudCommand(self, 'dataproc', 'jobs', 'submit', 'spark')
     cmd.flags['cluster'] = self.cluster_id
     cmd.flags['jar'] = jarfile
```

Comment:
Technically we suggest using --jars with --class, because --jar implies you are using the main method. This works fine mod an angry stderr message.